indigo_api/models/tasks.py

```diff
@@ -317,12 +317,8 @@ def update_blocked_tasks(self, task, user):
             # this task is no longer blocking other tasks
             task.blocking.clear()
             for blocked_task in blocked_tasks:
-                if has_transition_perm(blocked_task.unblock, user):
-                    # the other task no longer has blocking tasks; unblock it
-                    blocked_task.unblock(user)
-                else:
-                    action.send(user, verb='updated', action_object=blocked_task,
-                                place_code=blocked_task.place.place_code)
+                action.send(user, verb='partially unblocked', action_object=blocked_task,
```

Comment:
I don't mind what the verb used internally is, but when we display it, maybe phrasing like "resolved a task that was blocking this one" would be clearer? Also, include the `task` as the action's target, so we have that backward reference.

Reply:
OR `... resolved {task} that was blocking this task`.